stats/lib/app.js

```diff
@@ -0,0 +1,151 @@
+    await withFilter(request, reply, filter => {
+      return fetchParticipantChangeRates(pgPools, filter)
+    })
+  })
+  app.get('/participant/:address/scheduled-rewards', async (/** @type {RequestWithFilterAndAddress} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchParticipantScheduledRewards(pgPools, filter, request.params.address)
+    })
+  })
+  app.get('/participant/:address/reward-transfers', async (/** @type {RequestWithFilterAndAddress} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchParticipantRewardTransfers(pgPools, filter, request.params.address)
+    })
+  })
+  app.get('/miners/retrieval-success-rate/summary', async (/** @type {RequestWithFilter} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchMinersRSRSummary(pgPools, filter)
+    })
+  })
+  app.get('/miners/retrieval-timings/summary', async (/** @type {RequestWithFilter} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchMinersTimingsSummary(pgPools, filter)
+    })
+  })
+  app.get('/retrieval-result-codes/daily', async (/** @type {RequestWithFilter} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchDailyRetrievalResultCodes(pgPools, filter)
+    })
+  })
+  app.get('/retrieval-timings/daily', async (/** @type {RequestWithFilter} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchDailyRetrievalTimings(pgPools, filter)
+    })
+  })
+  app.get('/miner/:minerId/retrieval-timings/summary', async (/** @type {RequestWithFilterAndMinerId} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchDailyMinerRetrievalTimings(pgPools, filter, request.params.minerId)
+    })
+  })
+  app.get('/miner/:minerId/retrieval-success-rate/summary', async (/** @type {RequestWithFilterAndMinerId} */ request, reply) => {
+    await withFilter(request, reply, filter => {
+      return fetchDailyMinerRSRSummary(pgPools, filter, request.params.minerId)
+    })
+  })
+  app.get('/miner/:minerId/deals/eligible/summary', (request, reply) => {
+    redirectToSparkApi(request, reply, SPARK_API_BASE_URL)
+  })
+  app.get('/client/:clientId/deals/eligible/summary', (request, reply) => {
+    redirectToSparkApi(request, reply, SPARK_API_BASE_URL)
+  })
+  app.get('/allocator/:allocatorId/deals/eligible/summary', (request, reply) => {
+    redirectToSparkApi(request, reply, SPARK_API_BASE_URL)
+  })
+  addPlatformRoutes(app, pgPools)
+  app.get('/', (request, reply) => {
+    reply.send('OK')
+  })
+
+  return app
+}
+
+/**
+ * @param {Fastify.FastifyRequest} request
+ * @param {Fastify.FastifyReply} reply
+ * @param {string} SPARK_API_BASE_URL
+ */
+const redirectToSparkApi = (request, reply, SPARK_API_BASE_URL) => {
```

Comment:
It's a nitpick but I feel like this name could be more generic, maybe something like `redirectToURL` or something like that.

Reply:
But it's not that. A generic redirect would be `reply.redirect(url, status)`. This is a specific redirect to spark-api

Reply:
Fair enough, let's leave it as `redirectToSparkApi` then.